docs/user-manual/software/cltool.md

```diff
@@ -153,31 +154,71 @@ OPTIONS (RTK Rover / Base)
 Windows Visual Studio supports CMake projects. Follow the instructions provided by Microsoft:
 https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170
 
-## Update the Firmware
-In order to update the firmware of your unit on the CLTool, follow these steps:
+## Updating Firmware with CLTool
+### Updating using Firmware Package
 
-1. Navigate to the directory with the CLTool executable
-2. Set the unit's COM port as an option, e.g. `-c COM15`
-3. Specify the FILEPATH to the .hex file, e.g. `-uf foo/bar/IS_uINS-3.hex`
-4. Optionally specify the bootloader BLFILEPATH to the .bin file, e.g. `-ub foo/bar/bootloader-SAMx70.bin`
-5. Run the executable
+Updating firmware using a firmware package file provides a simple method to update multiple devices in one process.  This include the ability to update an IMX-GPX module pair in one step.  The cltool only needs know the file path of the firmware package file and the serial port of the device to be updated.  The file extension for a firmware package is `.fpkg`.
```

Comment:
This suggests that the IMX can be updated, currently.  It cannot.  Perhaps rather than saying IMX-GPX, reference GPX and the GNSS receivers?  Otherwise, we should note that the IMX is not currently supported.